extension/content_script.js

```diff
@@ -251,11 +256,21 @@ function parseDocument() {
 }
 
 let readableDomIds = parseDocument();
+doc = new Doc(readableDomIds);
 tracker = new Tracker(readableDomIds);
-display = new Display(readableDomIds, speed);
+window.tracker = tracker; 
+display = new Display(readableDomIds, speed, doc.getTotalWords());
+window.display = display;
+
+
+
 setupClickListener(tracker);
 readListener();
 
+startMove(direction.FORWARD); // Start reader on the first line
```

Comment:
[just a comment, please resolve] --> in google we have a way of indicating these (comments that don't have to be resolved), but idk how to in github :(
ah, a behavior change. i agree w/ this.